packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -36,9 +37,22 @@ export class JaegerExporter implements SpanExporter {
     this._onShutdownFlushTimeout =
       typeof config.flushTimeout === 'number' ? config.flushTimeout : 2000;
 
-    config.host = config.host || process.env.JAEGER_AGENT_HOST;
+    // https://github.com/jaegertracing/jaeger-client-node#environment-variables
+    // By default, the client sends traces via UDP to the agent at localhost:6832. Use JAEGER_AGENT_HOST and
+    // JAEGER_AGENT_PORT to send UDP traces to a different host:port. If JAEGER_ENDPOINT is set, the client sends traces
+    // to the endpoint via HTTP, making the JAEGER_AGENT_HOST and JAEGER_AGENT_PORT unused. If JAEGER_ENDPOINT is secured,
+    // HTTP basic authentication can be performed by setting the JAEGER_USER and JAEGER_PASSWORD environment variables.
+    if (config.endpoint) {
```

Comment:
for me it looks a bit strange that in order to use process.env you need to declare again an endpoint in config
I think this line should look like this:
```
if (config.endpoint || process.env.JAEGER_ENDPOINT) 
```

Reply:
@vmarchaud alluded to this in a previous comment.

Reply:
@dyladan I see it now, but pls correct me if I'm wrong - it hasn't been fixed ?

Reply:
it has not been fixed

Reply:
This issue is still not fixed